python/lsst/ip/isr/ampOffset.py

```diff
@@ -179,6 +202,9 @@ def run(self, exposure):
                 "All pixels masked: cannot calculate any amp offset corrections. All pedestals are being set "
                 "to zero."
             )
+            print(
```

Comment:
Why is there a `print` statement here? The log message above should be sufficient? A task should not normally be making use of `print` in production code.